integration

```diff
@@ -619,6 +619,7 @@
   "mikelawrence/senseme-hacs",
   "mill1000/midea-ac-py",
   "MislavMandaric/home-assistant-vaillant-vsmart",
+  "mlamberts78/weather-chart-card",
```

Comment:
This is not an integration.

Reply:
Indeed chnaged the wrong file, should be plugin.